config/testdata/conf.good.yml

```diff
@@ -109,7 +109,8 @@ scrape_configs:
 - job_name: service-kubernetes
 
   kubernetes_sd_configs:
-  - api_servers:
+  - kind: service_endpoints
```

Comment:
Agree it's confusing, such overloaded terms. Currently it's `endpoint` & `service` which is consistent with Kubernetes naming which is probably the best way to go.